boards/wm1110dev/src/main.rs

```diff
@@ -275,6 +288,48 @@ pub unsafe fn start() -> (
     components::debug_writer::DebugWriterComponent::new(uart_mux)
         .finalize(components::debug_writer_component_static!());
 
+    //--------------------------------------------------------------------------
+    // SENSORS
+    //--------------------------------------------------------------------------
+
+    let _ = &nrf52840_peripherals.gpio_port[I2C_PWR].make_output();
+    let _ = &nrf52840_peripherals.gpio_port[I2C_PWR].set();
+
+    let sensors_i2c_bus = static_init!(
+        capsules_core::virtualizers::virtual_i2c::MuxI2C<'static, nrf52840::i2c::TWI>,
+        capsules_core::virtualizers::virtual_i2c::MuxI2C::new(&base_peripherals.twi1, None,)
+    );
+    kernel::deferred_call::DeferredCallClient::register(sensors_i2c_bus);
+    base_peripherals.twi1.configure(
+        nrf52840::pinmux::Pinmux::new(I2C_SCL_PIN as u32),
+        nrf52840::pinmux::Pinmux::new(I2C_SDA_PIN as u32),
+    );
+    base_peripherals.twi1.set_master_client(sensors_i2c_bus);
```

Comment:
Can you use the i2c mux component instead?

https://github.com/tock/tock/blob/3a0527d586702b8ae8cb242391fa72eb3a412f49/boards/components/src/i2c.rs#L69